src/Apps/Artist/Routes/AuctionResults/__tests__/ArtistAuctionResults.jest.tsx

```diff
@@ -102,13 +102,7 @@ describe("AuctionResults", () => {
       mockUseAuthDialog.mockImplementation(() => ({ showAuthDialog }))
 
       let operationVariables
-      const { env } = renderWithRelay(mockedResolver, true)
```

Comment:
This test is the only one that uses `manualEnvControl`. However, it doesn't seem actually necessary so I simplified it.